docs/source/contributor.rst

```diff
@@ -1,10 +1,155 @@
-===========
-Contributor
-===========
+.. _contributing:
 
-.. toctree::
-   :maxdepth: 1
-   :caption: Contributor Documentation
+Contributing to Jupyter Notebook
```

Comment:
How about modifying `CONTRIBUTING.md` directly? It is included in the docs automatically.

https://github.com/jupyter/notebook/blob/main/CONTRIBUTING.md